src/components/MeetupList.tsx

```diff
@@ -115,24 +47,63 @@ const MeetupList: React.FC<IProps> = () => {
     setSearchField(event.target.value)
 
   return (
-    <div>
-      <StyledInput
+    <Box>
+      <Input
         onChange={handleSearch}
         placeholder={"Search by meetup title or location"}
+        display="block"
+        mr="auto"
+        ml="auto"
+        mb={6}
+        border="1px solid"
+        borderColor="searchBorder"
+        color="text"
+        p={2}
+        borderRadius={1}
+        w="100%"
+        _focus={{ outline: "primary solid 1px" }}
+        _placeholder={{ color: "text200" }}
       />
-      <Table>
+      <Box boxShadow="tableBoxShadow">
```

Comment:
As mentioned in the prev PR, this `tableBoxShadow` shadow doesn't exist. You will need to create it in https://github.com/ethereum/ethereum-org-website/blob/dev/src/%40chakra-ui/gatsby-plugin/foundations/shadows.ts